docs/components/secnav/index.html

```diff
@@ -72,15 +72,15 @@ <h3>
                 in your organization to request access to this item.
               </hx-tooltip>
             </a>
-
+  
             <a href="#">
               <hx-icon type="account"></hx-icon>
               <p>Account</p>
             </a>
           </div>
           <div class="hxSpacer"></div>
           <div class="hxTopNavMenu">
-            <hx-disclosure aria-controls="demo-user-menu">
+            <hx-disclosure aria-controls="demo-user-menu" aria-expanded="true">
```

Comment:
Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

Reply:
Changes have been done, please review.